bls/bignum_pure.go

```diff
@@ -8,10 +8,7 @@ import (
 	"math/big"
 )
 
-var _modulus big.Int
-
 func init() {
```

Comment:
Same deal as above. The should be a build error once you enable the `bignum_pure` build tag.

Reply:
fixed